.gitignore

```diff
@@ -4,3 +4,5 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+assets/.DS_Store
+assets/images/.DS_Store
```

Comment:
You can just:
```suggestion
.DS_Store
```

Reply:
yeah or just add it to your global `~/.gitignore` not to the project one.